testfixture/testfixture.go

```diff
@@ -0,0 +1,33 @@
+package testfixture
```

Comment:
this name is too generic, try thinking of a name that better describes what this does.

Reply:
testfixture supposedly has generic shared libraries for the tests that we write. Unless we want to shrink only to what it does.

Reply:
This is like calling a library "utils" it makes it impossible for someone to know what is in it without reading all the code